matlab/msbuild/ice.proj

```diff
@@ -6,36 +6,30 @@
         <Configuration Condition="'$(Configuration)' == ''">Release</Configuration>
         <RootDir>$([System.IO.Path]::GetFullPath( $(MSBuildThisFileDirectory)..\msbuild ))</RootDir>
         <ToolboxDir>$(MSBuildThisFileDirectory)..\toolbox\build</ToolboxDir>
+        <MatlabVersion Condition="'$(MatlabVersion)' == ''">R2024a</MatlabVersion>
+        <MatlabHome Condition="'$(MatlabHome)' == ''">C:\Program Files\MATLAB\$(MatlabVersion)</MatlabHome>
+        <!-- Override with the env settings -->
+        <MatlabVersion Condition="'$(MATLAB_VERSION)' != ''">$(MATLAB_VERSION)</MatlabVersion>
```

Comment:
Should this go up on line 10?

Presumably if you set MATLAB_VERSION but not MATLAB_HOME, you want MatlabHome to be set correctly.

Reply:
Yes. Fixed